charts/flux-operator/values.yaml

```diff
@@ -60,6 +60,12 @@ serviceAccount: # @schema default: {"create":true,"automount":true,"name":""}
   automount: true
   name: ""
 
+rbac:
+   # -- Grant the cluster-admin role to the flux-operator service account (required for the Flux Instance deployment).
+  create: true
+  # -- Grant the Kubernetes view, edit and admin roles access to ResourceSet APIs.
+  createAggregation: true
```

Comment:
If the default for `create` is already true, granting `cluster-admin` by default, shouldn't this default of `createAggregation: true` be unnecessary? Can it not default to `false`?

Reply:
Oh wait this just creates some `ClusterRoles`, it doesn't create a `ClusterRoleBinding` for the flux-operator. I'm not sure I understand the documentation for this field then. What's the use case exactly?

Reply:
These are standard Kubernetes RBAC aggregation rules. Flux has them and any other controller usually.